packages/react/src/template.tsx

```diff
@@ -58,15 +61,17 @@ export type ResponseDataState = {
   actions: ActionDataState[]
 }
 export type HandlerReturnType = ResponseDataState | JSONValue | void
-export type HandlerFunction<T = any> = (args: T) => Promise<HandlerReturnType> | HandlerReturnType
+export type HandlerFunction<T extends any[] = any[]> = (
+  ...args: T
```

Comment:
We're changing payload signature from single value to an array of values, so that if function takes multiple arguments, we can pass them all to the server.